python/ray/dag/tests/test_accelerator_dag.py

```diff
@@ -0,0 +1,122 @@
+import pytest
```

Comment:
Rename this test file or combine with other test files? I don't think the tested APIs have anything to do with accelerators.